Formula/leela-zero.rb

```diff
@@ -0,0 +1,37 @@
+class LeelaZero < Formula
+  desc "Neural Network Go engine with no human-provided knowledge"
+  homepage "https://zero.sjeng.org/"
+  url "https://github.com/gcp/leela-zero/archive/v0.15.tar.gz"
+  sha256 "6a34df356cf64aaa74bdf90826f8653255c42c3e65da8715f9c39751de00926a"
+
+  option "without-network", "Don't automatically download the best known network"
+
+  depends_on "boost"
+
+  resource "network" do
+    url "http://zero.sjeng.org/best-network", :using => :nounzip
+  end
+
+  def install
+    cd "src" do
+      system "make"
+      bin.install "leelaz"
```

Comment:
No man page, no headers, no library to install? No `make install` target?

Reply:
Technically it also produces the `autogtp` binary for submitting training games to the project's distributed system, but I suspect 99% of Macs out there are not powerful enough to make a worthwhile contribution anymore (since the latest network size increase) so I deliberately left it out.

But no, no libraries or man pages beyond that.